analytics/views.py

```diff
@@ -1144,8 +1148,9 @@ def support(request: HttpRequest) -> HttpResponse:
                 context["error_message"] = error.message
             else:
                 do_change_realm_subdomain(realm, new_subdomain)
-                context["success_message"] = f"Subdomain changed from {old_subdomain} to {new_subdomain}"
-                context["redirect_to"] = "/activity/support?" + urlencode({"q": new_subdomain})
+                request.session["success_message"] = f"Subdomain changed from {old_subdomain} to {new_subdomain}"
+                redirect_url = f"/activity/support?q={new_subdomain}"
```

Comment:
`new_subdomain` still needs to be URI-escaped -- it's still in a URL, as the variable name you chose indicates. :)

Reply:
yes! thanks for pointing this out... made the change!